gha/jira/util.py

```diff
@@ -1,12 +1,22 @@
 import re
 from typing import Union
 
-JIRA_ISSUE_RE = re.compile(r"^GBL[A-Z]*-[0-9]+", re.IGNORECASE)
+JIRA_ISSUE_RE = re.compile(r"^(GBL[A-Z]*)-(\d+)", re.IGNORECASE)
 # Ex: com.atlassian.greenhopper.service.sprint.Sprint@604f8041[id=3823,rapidViewId=525,
 # state=ACTIVE,name=DevOps 2022/48,startDate=2022-11-28T23:19:00.000+03:00,
 # endDate=2022-12-02T23:19:00.000+03:00,completeDate=<null>,
 # activatedDate=2022-11-28T23:19:31.860+03:00,sequence=3823,goal=,autoStartStop=false
 AGILE_SPRINT_ID_RE = re.compile(r"\[id=(\d+),")
+TITLE_RE = re.compile(r'^(#)+\s', re.MULTILINE)
+
+
+def replace_title(matchobj):
+    return f'h{len(matchobj.group(0)) - 1}. '
```

Comment:
Нулевая группа - вся регулярка. Тебе нужна первая - захваченные "#" внутри первых скобок. -1 тебе вероятно понадобилась именно для компенсации ошибки.

```suggestion
    return f'h{len(matchobj.group(1))}. '
```